erpnext/stock/doctype/delivery_note_item/delivery_note_item.json

```diff
@@ -729,11 +730,17 @@
    "fieldtype": "Link",
    "label": "Package Tag",
    "options": "Package Tag"
+  },
+  {
+   "fieldname": "certificate_of_analysis",
+   "fieldtype": "Attach",
+   "label": "Certificate of Analysis"
```

Comment:
Don't allow users to attach directly to the Delivery Note, they should go through the Quality Inspection and Batch for that